From 34f4d3288bede886c599c235a728277d91955ac9 Mon Sep 17 00:00:00 2001
From: Stefan Lange <s.lange@gateware.de>
Date: Wed, 12 Oct 2016 13:41:40 +0200
Subject: [PATCH] TQMT1042: corrected error in reset configuration word

SRDS_PLL_REF_CLK_SEL_S1 PLL2 was accidentally set to 125 Mhz instead
of 100 MHz.
This resulted in the SATA lanes not working.

Signed-off-by: Stefan Lange <s.lange@gateware.de>
---
 board/tqc/tqmt1042/tqmt1042_serdes86_rcw.cfg | 2 +-
 board/tqc/tqmt1042/tqmt1042_serdes88_rcw.cfg | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/tqc/tqmt1042/tqmt1042_serdes86_rcw.cfg b/board/tqc/tqmt1042/tqmt1042_serdes86_rcw.cfg
index fa0a5f6..38597ae 100644
--- a/board/tqc/tqmt1042/tqmt1042_serdes86_rcw.cfg
+++ b/board/tqc/tqmt1042/tqmt1042_serdes86_rcw.cfg
@@ -2,6 +2,6 @@
 aa55aa55 010e0100
 # serdes protocol 0x86
 0c10000e 0e000000 00000000 00000000
-86000000 00000001 ec027000 01000000
+86000000 80000001 ec027000 01000000
 00000000 00000000 00000000 0002a800
 00000000 01fe5005 00000000 00000000
diff --git a/board/tqc/tqmt1042/tqmt1042_serdes88_rcw.cfg b/board/tqc/tqmt1042/tqmt1042_serdes88_rcw.cfg
index 1e06ad5..3c92767 100644
--- a/board/tqc/tqmt1042/tqmt1042_serdes88_rcw.cfg
+++ b/board/tqc/tqmt1042/tqmt1042_serdes88_rcw.cfg
@@ -2,6 +2,6 @@
 aa55aa55 010e0100
 # serdes protocol 0x86
 0c10000e 0e000000 00000000 00000000
-88000000 00000001 ec027000 01000000
+88000000 80000001 ec027000 01000000
 00000000 00000000 00000000 0002a800
 00000000 01fe5005 00000000 00000000
-- 
1.9.1

